oomact_ros/src/CalibrationNode.cpp

```diff
@@ -0,0 +1,119 @@
+#include <rosbag/bag.h>
+#include <ros/ros.h>
+
+#include <aslam/calibration/ros/RosInputProvider.h>
+#include <aslam/calibration/calibrator/CalibratorI.hpp>
+#include <aslam/calibration/input/InputProviderI.h>
+#include <aslam/calibration/data/ObservationManagerI.h>
+#include <aslam/calibration/model/FrameGraphModel.h>
+#include <aslam/calibration/model/Model.h>
+#include <aslam/calibration/model/PoseTrajectory.h>
+#include <aslam/calibration/model/fragments/So3R3Trajectory.h>
+#include <aslam/calibration/model/sensors/PoseSensor.hpp>
+#include <aslam/calibration/tools/SmartPointerTools.h>
```

Comment:
Not all headers are required. But we can filter that later. I've cleaned up the RosInputTest in that respect. We can compare later.